queue_services/entity-emailer/tests/unit/email_processors/test_filing_notification.py

```diff
@@ -67,7 +81,7 @@ def test_correction_incorporation_notification(app, session, status, has_name_ch
         email = filing_notification.process(
             {'filingId': filing.id, 'type': 'correction', 'option': status}, token)
         if status == 'PAID':
-            assert 'comp_party@email.com' in email['recipients']
```

Comment:
I only changed it here because before I changed anything the test was failing. I checked the conditions and it seemed to be a wrong asserting for 'comp_party@email.com'.